test/test_nn.py

```diff
@@ -8324,6 +8323,44 @@ def eval_constructor(*args, **kwargs):
         test_params['constructor'] = gen_eval_constructor(test_params['constructor'])
         test = NewModuleTest(**test_params)
         add_test(test, decorator)
+    if 'check_with_long_tensor' in test_params:
+        fullname = test_params.get('fullname', None)
+        if fullname:
+            test_params['fullname'] = fullname + '_with_long_tensor'
+        else:
+            desc = test_params.get('desc', None)
+            test_params['desc'] = 'with_long_tensor' if desc is None else desc + '_with_long_tensor'
+
+        def gen_long_tensor(size):
```

Comment:
this name is confusing, because the name implies it returns a long tensor but it actually returns a double tensor.

Also, why do you need to specify CPU?

Reply:
Discussed offline, going to rename to `double_equivalent_of_long_tensor`.

Reply:
Done